### DISCUSSION TITLE
Concurrent scheduling/editing of sessions/releasing

### DISCUSSION THREAD

**Discussion by johanvdw** [pretalx/pretalx#1441] [General]
At FOSDEM we are trying to see if we can make pretalx replace our venerable pentabarf instance.

One of the main questions that remain for us is how we can handle concurrent editing of sessions. Eg the last edition we had 63 tracks, and those are usually managed by different people (eg the emulator devroom manager will only schedule his track), and scheduling typically happens in a short timespan. 

On the other hand, close to the actual conference, less changes are made, but these should be released immediately.

I think we should probably make release automatically (definitely close to the conference), but I'd be happy to hear if anyone else has had similar issues and how they solved it.

**Comment by rixx** [comment]
The schedule editor is kiiinda concurrent (you see changes others make with a lag of up to 10 seconds, it's old-style long-polling, but we don't plan to switch to more concurrent methods as the effort would exceed the usability gain). Automated releases do sound interesting, but should wait until we're done with the schedule editor rework (and then probably as a separate issue).
